runtime/rococo/src/lib.rs

```diff
@@ -1633,7 +1623,13 @@ pub mod migrations {
 	}
 
 	/// Unreleased migrations. Add new ones here:
-	pub type Unreleased = ();
+	pub type Unreleased = (
+		SetStorageVersions,
+		// Remove UMP dispatch queue <https://github.com/paritytech/polkadot/pull/6271>
+		parachains_configuration::migration::v6::MigrateToV6<Runtime>,
+		ump_migrations::UpdateUmpLimits,
+		pallet_society::migrations::MigrateToV2<Runtime, (), ()>,
```

Comment:
Shouldn't this PR only add this one migration? Why do you also add the others? Merge issue?